src/bandersnatch_filter_plugins/filename_name.py

```diff
@@ -16,6 +17,24 @@ class ExcludePlatformFilter(FilterReleaseFilePlugin):
     _patterns: List[str] = []
     _packagetypes: List[str] = []
 
+    _pythonversions = [
+        "py2.4",
+        "py2.5",
+        "py2.6",
+        "py2.7",
+        "py3.0",
+        "py3.1",
+        "py3.2",
+        "py3.3",
+        "py3.4",
+        "py3.5",
+        "py3.6",
+        "py3.7",
+        "py3.8",
+        "py3.9",
+        "py3.10",
+    ]
+
```

Comment:
Wonder if there is any way we can dynamically generate this when new versions come out :|
- Probably not worth the effort or complexity ..

(I just like to try and avoid things that need to be updated every new python release as it becomes operational overhead)

This is outside the scope of this PR, I'll make another issue for it when we merge, unless someone reading comes up with a clever idea.

Reply:
Here's a sample of a way to generate new versions.
Someone would still have to increment the major/minor versions, but its better than having to increment the list by hundreds (e.g. 3.98, 3.99) if it ever comes to that.
This assumes that Python never changes their version scheming too.

```python
_versions: List[str] = []
_ver_separator = '.'

_py_mjr_ver_min = 2
_py_mjr_ver_max = 3
_py_mjr_vers = list(map(str, list(range(_py_mjr_ver_min, _py_mjr_ver_max+1))))

_py2_mnr_ver_min = 4
_py2_mnr_ver_max = 7

_py3_mnr_ver_min = 0
_py3_mnr_ver_max = 10

_py2_mnr_vers = list(map(str, list(range(_py2_mnr_ver_min, _py2_mnr_ver_max+1))))
_py3_mnr_vers = list(map(str, list(range(_py3_mnr_ver_min, _py3_mnr_ver_max+1))))

_conf_prefix = 'py'

for mjr in _py_mjr_vers:
    if "2" == mjr:
        _mnr_versions = [_conf_prefix + mjr + _ver_separator + mnv_ver for mnv_ver in _py2_mnr_vers]
    elif "3" == mjr:
        _mnr_versions = [_conf_prefix + mjr + _ver_separator + mnv_ver for mnv_ver in _py3_mnr_vers]
    
    _versions.extend(_mnr_versions)


print(_versions)
# ['py2.4', 'py2.5', 'py2.6', 'py2.7', 'py3.0', 'py3.1', 'py3.2', 'py3.3', 'py3.4', 'py3.5', 'py3.6', 'py3.7', 'py3.8', 'py3.9', 'py3.10']

```